xla/stream_executor/gpu/gpu_command_buffer_test.cc

```diff
@@ -31,7 +32,24 @@ limitations under the License.
 #include "tsl/platform/test.h"
 #include "tsl/platform/test_benchmark.h"
 
-namespace stream_executor::cuda {
+namespace stream_executor::gpu {
+
+static Platform* GpuPlatform() {
+  auto name = absl::AsciiStrToUpper(xla::PlatformUtil::CanonicalPlatformName("gpu").value());
```

Comment:
what's the type of `name`? if it's `std::string` it will not pass checks internally, all static variables should be trivially destructible

Reply:
Not sure I follow this one?

Reply:
ah, my bad, I though `name` is a static variable.